037-WindowsVirtualDesktop/Student/10-Monitor-Manage-Performance-Health.md

```diff
@@ -0,0 +1,47 @@
+# Challenge 10: Monitor and manage the performance and health of your WVD environment
+
+[< Previous Challenge](./09-Automate-WVD-Tasks.md) - **[Home](../README.md)** - [Next Challenge>](./11-Configure-User-Experience-Settings.md)
+
+## Introduction
+
+You've recently deployed WVD for your company and each region started off ok however users have begun reporting problems occasional issues with connecting and performance has become progressively worse for some of the sessions. You need to offer a way to gather information to troubleshoot these issues and because they happen infrequently and across a range of users and times you need to be able to efficiently capture and record these events. It's important that you also can be more proactive to issues in the environment before they cause too much impact on the user experience. Your manager is also concerned about the cost of running so many WVD hosts across each of the regions when staff are not working and VMs may be sitting idle and wants to look at ways to reduce costs without negatively impacting the users when they login each morning. Lastly many users wish to use Microsoft Teams when connected to WVD hosts and use audio and video for their online meetings and although it works, users report regular stuttering during meetings and have asked whether it's possible to improve the experience.
+
+## Description
+
+The goal of this challenge is to demonstrate how you can enable a monitoring solution for WVD and how it may be used to identify and diagnose issues across your deployment. Using this information you should be able to pull in WVD service information, performance metrics and event log information that relates to the service and present it within the WVD insights dashboard. Your environment should also be enabled for automatic scaling based on your companies requirements to optimise costs and ensure sufficient capacity per your peak hour requirements. 
+
+**NOTE:** Scaling Automation was deployed in Challenge 9
+
+#### WVD Monitoring
```

Comment:
I would make these subsections under 'Description' 3 level headers, not 4 level headers because Description is a 2 level header.